.github/workflows/pull-request-checks.yaml

```diff
@@ -418,6 +418,52 @@ jobs:
       - name: Run tests
         run: cd build; ctest . -V -L CORE -j2
 
+  # This job takes approximately X to Y minutes
```

Comment:
Reminder to update, something close to 60 by the looks of the last run.

Reply:
Done, thank you for your careful review!